extensions/tekton/tests/src/test/java/io/fabric8/tekton/pipeline/v1beta1/TaskTest.java

```diff
@@ -32,7 +32,7 @@ void testTaskFromJSON() throws IOException {
     ObjectMapper mapper = new ObjectMapper();
     Task task = mapper.readValue(is, Task.class);
     assertEquals("Task", task.getKind());
-    assertEquals("tekton.dev/v1beta1", task.getApiVersion());
+    assertEquals("tekton.dev/v1", task.getApiVersion());
```

Comment:
This doesn't look okay to me. Could you please create a separate test for v1 Task?

Reply:
Ah yes, this is still in the `v1beta1` namespace, that's no good. Should I duplicate this test for v1 and leave the v1beta1 one as-is?

Reply:
I restored this as-is and make a new test for the `v1` one